scripts/tools/flip_video.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+This script helps to flip videos horizontally to create videos of opposite side.
```

Comment:
```suggestion
This script helps to flip videos horizontally for data augmentation. 

Generally, it can be used to quickly double the size of your dataset, or, in the case where you've collected data for an action performed on a specific side, you can flip these videos and use them to classify the opposite side.
```